webapp/views.py

```diff
@@ -914,3 +914,22 @@ def thank_you():
     return flask.render_template(
         "thank-you.html", referrer=flask.request.args.get("referrer")
     )
+
+
+def subscription_centre():
+    leadId = flask.request.args.get("id")
+    try:
+        response = marketo_api.request(
+            "GET", 
+            f"/rest/v1/lead/{leadId}.json", 
+            {"fields": "prototype_interests,email"}
+        )
+        data = response.json()
+    except Exception:
```

Comment:
maybe you want to catch the error `except Exception as error`

Reply:
This would show up in Sentry as a generic exception, but it won't tell you the details. You can also use a more granular Sentry exception with something [like this](https://github.com/canonical/ubuntu.com/blob/005db960881855b379b429f37f0edca91ffa469b/webapp/views.py#L888) or [this](https://github.com/canonical/ubuntu.com/blob/005db960881855b379b429f37f0edca91ffa469b/webapp/views.py#L120) which will throw that specific `HTTPerror`

Reply:
I have updated with your suggestion, can you take another look please